cmd/component-codegen/cmd/generator/generator.go

```diff
@@ -0,0 +1,42 @@
+package generator
+
+import (
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	cmdDir       = "cmd/component-codegen"
+	ApisDir      = "apis/components/v1alpha1"
+	Controllers  = "controllers/components"
+	DscApiDir    = "apis/datasciencecluster/v1/datasciencecluster_types.go"
```

Comment:
```suggestion
	DscApiFilePath    = "apis/datasciencecluster/v1/datasciencecluster_types.go"
```

^or something. I think the `Dir` suffix is probably not optimal, given that it's not a directory :slightly_smiling_face:

Reply:
Thanks for pointing this out. Changed now